api/src/modules/project/project.controller.ts

```diff
@@ -1,13 +1,23 @@
-import { Controller, Get } from '@nestjs/common';
-import { Project } from './project.model';
+import { Body, Controller, Get, Param, Post, Put } from '@nestjs/common';
+import { ProjectDto } from './project.dto';
 import { ProjectService } from './project.service';
 
 @Controller('project')
 export class ProjectController {
   constructor(private readonly projectService: ProjectService) {}
 
   @Get()
-  getProjects(): Project[] {
-    return this.projectService.getProjects();
+  async getAllProjects(): Promise<ProjectDto[]> {
+    return await this.projectService.getAllProjects();
+  }
+
+  @Get(':projectName')
```

Comment:
Did we decide to change this to `projectId`?

Reply:
done ✅